packages/react/lib/fragments.js

```diff
@@ -1,13 +1,6 @@
 const { renderToString } = require('react-dom/server');
-const {
-  Helmet: { renderStatic },
-} = require('react-helmet');
 
 module.exports = (element) => {
   const reactMarkup = renderToString(element);
-  const fragments = Object.entries(renderStatic()).reduce(
-    (result, [key, value]) => ({ ...result, [key]: value.toString() }),
-    { reactMarkup, headPrefix: '', headSuffix: '' }
-  );
-  return fragments;
+  return { reactMarkup };
```

Comment:
Should we delete this function / file?
We could potentially also delete the `renderToFragments` hook, because it is no longer needed (only hops-graphql relies on it and it also is not documented).

Reply:
We can definitely delete this function, but aren't we relying on the hook in [Apollo SSR](https://github.com/xing/hops/blob/6e5adbcdcbb6c54a9c2bb03dff1965cb9d0a3e7f/packages/graphql/mixin.server.js#L98)?

Reply:
It is also being imported in Hops: https://github.com/xing/hops/blob/6e5adbcdcbb6c54a9c2bb03dff1965cb9d0a3e7f/packages/graphql/mixin.server.js#L2 Let's leave it in for now, and discuss if/how we want to change the mixins in a different PR.